src/components/LastAction.tsx

```diff
@@ -0,0 +1,46 @@
+import type { FC } from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+
+import { actions as hijackedActions } from "../redux/actions";
+import { selectLastAction } from "../redux/actionsSlice";
+import { actions as loadingActions } from "./TodoList/actions";
+
+const loading = [loadingActions.LOADING_STARTED, hijackedActions.QUERY_PENDING, hijackedActions.MUTATION_PENDING];
```

Comment:
Tohle je obecně docela zvláštní koncept, možná bych se o tom pak pobavil osobně.

Reply:
Znova len veľmi hacky riešenie aby som docielil to čo som písal v komentári vyššie.